_includes/gpu-labels-table.html

```diff
@@ -11,5 +11,7 @@
     {% include gpu-labels-table-row.html arch="amd64" gpu="V100" driver="450" latest=false %}
     {% include gpu-labels-table-row.html arch="amd64" gpu="V100" driver="520" latest=true %}
     {% include gpu-labels-table-row.html arch="arm64" gpu="A100" driver="520" latest=true %}
+    {% include gpu-labels-table-row.html arch="amd64" gpu="T4" driver="520" latest=true %}
```

Comment:
```suggestion
    {% include gpu-labels-table-row.html arch="amd64" gpu="T4" driver="520" latest=false %}
```

As we mentioned in Slack, let's omit the `latest` tag from T4s for now.